src/t_stream.c

```diff
@@ -2466,16 +2644,43 @@ NULL
     }
 }
 
-/* XSETID <stream> <id>
+/* XSETID <stream> <id> [offset xdel_max_id]
```

Comment:
please mention the new argument for XSETID in the top comment, and also mention it is used in propagation (AOF / replicas)

Reply:
Done.